by-language/python-sqlalchemy/async_streaming.py

```diff
@@ -0,0 +1,168 @@
+"""
+About
+=====
+
+Example program to demonstrate how to connect to CrateDB using its SQLAlchemy
+dialect, and exercise a few basic examples using the low-level table API, in
+asynchronous mode.
+
+Specific to the asynchronous mode of SQLAlchemy is the streaming of results:
+
+> The `AsyncConnection` also features a "streaming" API via the `AsyncConnection.stream()`
+> method that returns an `AsyncResult` object. This result object uses a server-side cursor
+> and provides an async/await API, such as an async iterator.
+>
+> -- https://docs.sqlalchemy.org/en/20/orm/extensions/asyncio.html#synopsis-core
+
+Both the PostgreSQL drivers `asyncpg` and `psycopg` can be used.
+The corresponding SQLAlchemy dialect identifiers are::
+
+    # PostgreSQL protocol on port 5432, using `asyncpg`
+    crate+asyncpg://crate@localhost:5432/doc
+
+    # PostgreSQL protocol on port 5432, using `psycopg`
+    crate+psycopg://crate@localhost:5432/doc
+
+
+Synopsis
+========
+::
+
```

Comment:
Perhaps we could add around here the dependencies as well?
ex: 
`pip install sqlalchemy psycopg2..`

Reply:
Currently, `asyncpg` and `psycopg`(3) are pulled in by [sqlalchemy-postgresql-relaxed](https://github.com/pyveci/sqlalchemy-postgresql-relaxed/blob/0.1.1/pyproject.toml#L59-L63). If you think it should be done differently, either omitting them there, or making the spot more selectively by introducing extras, let me know.